devices.js

```diff
@@ -9119,6 +9119,23 @@ const devices = [
         fromZigbee: [fz.on_off],
         toZigbee: [tz.on_off],
     },
+    {
+        zigbeeModel: ['DM2500ZB'],
+        model: 'DM2500ZB',
+        vendor: 'Sinope',
+        description: 'Zigbee smart dimmer',
+        extend: generic.light_onoff_brightness,
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(1);
+            const binds = [
+                'genBasic', 'genIdentify', 'genGroups', 'genOnOff', 'genLevelCtrl',
```

Comment:
`genBasic`, `genIdentify`, `genGroups` can be removed here.

Reply:
Done.

Reply:
Should I squash both commit + force push? I'm not familiar with Github pull-request workflow.

Reply:
Not needed, I can squash it from my side, thanks!